pkgs/development/python-modules/schemdraw/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromBitbucket
+, pyparsing
+, matplotlib
+, latex2mathml
+, ziafont
+, ziamath
+, pytest
+, pytest-cov
+, nbval
+}:
+
+buildPythonPackage rec {
+  pname = "schemdraw";
+  version = "0.16";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromBitbucket {
+    owner = "cdelker";
+    repo = pname;
+    rev = "${version}";
+    hash = "sha256-W9sXtYI8gEwQPRo50taEGT6AQG1tdAbeCtX49eHVvFQ=";
+  };
+
+  propagatedBuildInputs = [
+    pyparsing
+  ];
+
+  passthru.optional-dependencies = {
+    matplotlib = [
+      matplotlib
+    ];
+    svgmath = [
+      latex2mathml
+      ziafont
+      ziamath
+    ];
+  };
+
+  nativeCheckInputs = [
+    pytest
+    pytest-cov
+    nbval
+    matplotlib
+    latex2mathml
+    ziafont
+    ziamath
+  ];
+
+  # Strip out references to unfree fonts from the test suite
+  postPatch = ''
+    substituteInPlace test/test_styles.ipynb --replace "font='Times', " ""
+  '';
+
+  checkPhase = ''
+    sh ./runtest.sh
```

Comment:
Please do not use peoples wrappers scripts. We want to execute pytest through our hook